extensions/skins/conversation_v1/Conversation.js

```diff
@@ -523,6 +526,18 @@ oppia.directive('conversationSkin', [function() {
         }, true);
       };
 
+      $scope.submitAnswer = function(answer) {
+        var timeAtSubmission = new Date().getTime();
```

Comment:
Seems like you want to set this only after we've checked that there's no duplicate submission in process